Rakefile

```diff
@@ -9,6 +9,7 @@ Rake::TestTask.new do |t|
   t.libs << 'test'
   t.test_files = FileList['test/**/*_test.rb']
   t.verbose = true
+  t.warning = false
```

Comment:
Rouge is spitting out a ton of warnings which made the test output a bit annoying, so I added this flag.